packages/docs/src/page-configs/ui-elements/dropdown/examples/PlacementAndOffset.vue

```diff
@@ -57,6 +56,7 @@ export default {
   },
 
   placement: computed(() => {
+    // eslint-disable-next-line no-undef
```

Comment:
Hi, @rustem-nasyrov, @aluarius, importing `aliasToPlacement` cause `yarn serve:docs` to error, without importing I can't commit with git.

Reply:
Hi, @xiongmao86, have you tried to import like this: `import { useColors } from 'vuestic-ui/src/composables'`

Reply:
Thank you, @rustem-nasyrov, I make a mistake that expose `aliasToPlace` directly to `index.ts` of `composables`.